credentials/apps/verifiable_credentials/models.py

```diff
@@ -0,0 +1,54 @@
+"""
+Database models for verifiable_credentials.
+"""
+from config_models.models import ConfigurationModel
+from django.conf import settings
+from django.contrib.sites.models import Site
+from django.db import models
+from organizations.models import Organization
+
+from .constants import OPEN_BADGES_V3_KEY, VERIFIABLE_CREDENTIAL_KEY
+
+
+class IssuanceConfiguration(ConfigurationModel):
+    """
+    A model representing the configuration settings for digital credentials issuance.
+    This model stores the details needed to generate and issue digital credentials, such as credential type, site, etc.
+
+    .. no_pii:
+    """
+
+    KEY_FIELDS = (
+        "site_id",
+        "slug",
+    )
+    DIGITAL_CREDENTIAL_TYPE_CHOICES = (
+        (VERIFIABLE_CREDENTIAL_KEY, "Verifiable Credentials"),
+        (OPEN_BADGES_V3_KEY, "Open Badges v3"),
+    )
+
+    enabled = models.BooleanField(default=False)
+    site = models.ForeignKey(
+        Site,
+        default=settings.SITE_ID,
+        related_name="%(class)ss",
+        on_delete=models.CASCADE,
+    )
+    slug = models.SlugField(
+        max_length=30,
+        default="default",
+        blank=True,
+    )
+    organization = models.ForeignKey(
```

Comment:
Did you see [this](https://github.com/raccoongang/credentials/blob/dcc.master/credentials/apps/catalog/models.py#L17)?